db/migrate/20210414171109_add_unit_value_constraint.rb

```diff
@@ -0,0 +1,10 @@
+class AddUnitValueConstraint < ActiveRecord::Migration[5.0]
+  def up
+    execute "UPDATE spree_variants SET unit_value = 1 WHERE unit_value <= 0"
```

Comment:
Just out of curiosity, how many do we have in this state?

Reply:
UK had 381 variants with unit_value == 0

Reply:
ah well, not that bad.